pkg/containers/containers.go

```diff
@@ -17,16 +17,12 @@ import (
 	"github.com/aquasecurity/libbpfgo"
 	"github.com/aquasecurity/tracee/pkg/cgroup"
 	cruntime "github.com/aquasecurity/tracee/pkg/containers/runtime"
-	"github.com/aquasecurity/tracee/pkg/mount"
 )
 
-var cgroupV1HierarchyID int
-
 // Containers contains information about running containers in the host.
 type Containers struct {
-	cgroupV1   bool
-	cgroupMP   string
-	cgroups    map[uint32]CgroupInfo
+	cgroups    *cgroup.Cgroups
+	cgroupsMap map[uint32]CgroupInfo
```

Comment:
suggestion: maybe `cgroupsInfo` or `cgroupsCache`? WDYT?

Reply:
Nope, Info is already used and it is not a cache =).